proof/refine/ARM/Tcb_R.thy

```diff
@@ -279,12 +280,11 @@ lemma readreg_corres:
         apply wp+
       apply (rule corres_when [OF refl])
       apply (rule suspend_corres)
-     apply wp+
-   apply (clarsimp simp: invs_def valid_state_def valid_pspace_def
+     apply (wp IpcCancel_AI.suspend_invs)+
```

Comment:
Maybe it's best to rename `susped_invs'` (I know there are a bunch of lemmas where we have written `invs` instead of `invs'`. We could make a good ol' JIRA issue out of it)

Reply:
`suspend_invs'` doesn't exist here yet, so it's enough to just remove the `IpcCancel_AI.`

Reply:
A JIRA issue for consistent naming sounds good to me!

Reply:
For this case it was actually the AInvs `invs`, I think. For what Michael’s saying, could you write it up as a JIRA issue, Michael, do you think?